docs/deploying/deploy-smart-contracts.mdx

```diff
@@ -8,9 +8,9 @@ To deploy your smart contracts to a live network, there are a few things you nee
 
 ## 1. Select the network
 
-By default, `yarn deploy` will deploy the contract to the local network. You can change the defaultNetwork in `packages/hardhat/hardhat.config.ts.` You could also simply run `yarn deploy --network target_network` to deploy to another network.
+By default, `yarn deploy` will deploy the contract to the local network. You can change the defaultNetwork in `packages/hardhat/hardhat.config.ts.` or `packages/foundry/foundry.toml` depending if you are using Hardhat or Foundry. You could also simply run `yarn deploy --network target_network` to deploy to another network.
```

Comment:
I think we can remove the deployment part since its mentioned below

```suggestion
By default, `yarn deploy` will deploy the contract to the local network. You can change the defaultNetwork in `packages/hardhat/hardhat.config.ts.` or `packages/foundry/foundry.toml` depending if you are using Hardhat or Foundry.
```